botto/config.py

```diff
@@ -176,7 +176,7 @@ def parse(config):
             "please": {
                 "trigger": "^pl(?:e+)ase",
                 "reactions": ["🥺"],
-                "exclude_guilds": ["833842753799848016"],
+                "exclude_guilds": ["833842753799848016", "890978723451523083"],
```

Comment:
Did you intend to disable this here? Or in the sorry section?

Reply:
I added the "sorry" reaction as a option for a "disabled feature" (making it remotely configurable). Whilst I was in there, I remembered this reaction and disabled this as well; this one is a "pattern" reaction where the exclusions are currently hardcoded.